bylaws/003-votes.md

```diff
@@ -1,12 +1,14 @@
 # Votes
 
-There are a number of different types of vote, each of which has its own rule set and expected threshold.  Unless otherwise specified, all votes take place over a period of two weeks, or until all eligible voters have voted, whichever comes first and votes are public.  Explicit abstentions (+0) will count toward quorum, if applicable, but do not count toward majority.
+There are a number of different types of vote, each of which has its own rule set and expected threshold. Unless otherwise specified, all votes take place over a period of two weeks, or until all eligible voters have voted, whichever comes first, and votes are public. Explicit abstentions (+0) will count toward quorum, if applicable, but do not count toward majority.
 
 The mechanics of voting (email, forum, any conventional subject lines, etc.) may be established and publicized by the Secretaries.
 
 Votes may be changed at will until the end of the voting period.
 
-In some cases, votes may overlap such that an individual would become eligible to cast a vote only as a result of another overlapping vote, such as a membership vote.  In all cases, eligibility to vote will be defined based on membership at the moment a given vote starts.  For nominations for Secretary or Core Committee member, eligibility to vote will be defined based on membership at the moment the nominating period begins.
+In some cases, votes may overlap such that an individual would become eligible to cast a vote only as a result of another overlapping vote, such as a membership vote. In all cases, eligibility to vote will be defined based on membership at the moment a given vote starts. For nominations for Secretary or Core Committee member, eligibility to vote will be defined based on membership at the moment the nominating period begins.
+
+A secretary can trigger any type of vote.
```

Comment:
The reason I added it here is because I removed it from the mention of specific votes below.